synthtool/gcp/common.py

```diff
@@ -153,10 +153,22 @@ def _load_partials(self, metadata: Dict):
 
 def decamelize(str: str):
```

Comment:
`str` is a built-in, so we probably shouldn't use it as a variable name. How about `value` or `str_`?